src/amo/reducers/ratings.js

```diff
@@ -4,18 +4,20 @@ export const initialState = {
   userRatings: {},
```

Comment:
huh. I just noticed that `userRatings` is redundant since this already gets reduced to the `ratings` key in the global state. I'll remove it.